packages/fxa-auth-server/scripts/move-customers-to-new-plan-v2/move-customers-to-new-plan-v2.ts

```diff
@@ -0,0 +1,273 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Stripe from 'stripe';
+import { writeFile } from 'fs/promises';
+import PQueue from 'p-queue';
+import { PayPalHelper } from '../../lib/payments/paypal';
+
+import { ACTIVE_SUBSCRIPTION_STATUSES } from 'fxa-shared/subscriptions/stripe';
+import { RefundType } from '@fxa/payments/paypal';
+
+export class CustomerPlanMover {
+  private stripeQueue: PQueue;
+
+  constructor(
+    private sourcePlanId: string,
+    private destinationPlanId: string,
+    private excludeCustomersHavingPriceIds: string[],
+    private outputFile: string,
+    private stripe: Stripe,
+    public dryRun: boolean,
+    rateLimit: number,
```

Comment:
I don't see this being set or accessed outside of the constructor, I think this can be made private.
```suggestion
    private rateLimit: number,
```